python/cudf/cudf/_cuda/gpu.pyx

```diff
@@ -367,12 +372,17 @@ def deviceGetName(int device):
     and status code.
     """
 
+    cdef CUresult init_status = cuInit(0)
```

Comment:
This is setting a bad precedent that we're going to need to add a `cuInit` call in every driver API wrapper we build.

Should we just initialize in an `__init__.py(x)` somewhere?

Reply:
Apparently `cuInit` only incurs a cost once (based on asking internally).

That said, I don't have strong feelings on how we do this. Though it does appear we do need to do this for the Driver API calls to work correctly.

Reply:
Couldn't we just call a `cuInit` within the Cython module `__init__.pyx` file to guarantee the driver is initialized before any function can be used?

Reply:
I'd rather not slow down `import` time further. Outside of that I don't really have any preferences.

Reply:
> I'd rather not slow down `import` time further. Outside of that I don't really have any preferences.

We're already calling runtime APIs on import that implicitly initialize the driver (without creating a context), so this shouldn't change import time.